sdk/maps/maps-route-rest/swagger/README.md

```diff
@@ -29,7 +31,7 @@ rest-level-client: true
 security: AzureKey
 security-header-name: subscription-key
 use-extension:
-  "@autorest/typescript": "6.0.0-rc.3"
+  "@autorest/typescript": "latest"
```

Comment:
Could you pin the version? otherwise we may not know what specific version if debugging in future?